### ISSUE TITLE
Have you successfully trained the network?

### ISSUE THREAD

**Issue by tangyuxing** [TRKuan/cxr8#1]
Hi TRKuan,

This is a good repo! Have you successfully trained the network?

I tried the training code you provided, the loss is almost 0 and the AUC is around 0.5. It seems that the network is not learning.

YT

**Comment by TRKuan** [comment]
Hi tangyuxing,

The code is kind of messy here. I'm still working on it. I just start learning deep learning so I'm trying lots of different things and didn't make the code nice and clean.

I trained with train.py and the AUC is about 0.75 on validation with very serous overfitting. Other scripts seems to have bugs and I'm trying to fix them but I just too busy right now.

Thanks for the comment

TR

**Comment by tangyuxing** [comment]
Thanks for the response! I will keep watching the update of this repo.

**Comment by malgamves** [comment]
Hey, 

I'm interested in the progress of the training. How is it going?

**Comment by TRKuan** [comment]
Well, actually, I'm not working on this project right now. I kind of can't figure out how to deal with it so I've moved to other project.
Sorry for the late replay.

**Comment by malgamves** [comment]
I might fork this and try and get it going. Will let you how it goes.

**Comment by PoonamZ** [comment]
@malgamves have you tried this repository.

**Comment by malgamves** [comment]
Hi, @PoonamZ no I haven't, I did a [similar implementaion](https://github.com/malgamves/DeepClassifyML) on my own though. Badly documented at the moment, working on that.

**Comment by PoonamZ** [comment]
@malgamves OK.
